test/test_unresolved_links.cpp

```diff
@@ -70,16 +70,80 @@ TEST(Links_UnresolvedBasic)
     // FIXME: CHECK_EQUAL(q.count(), 1);
 
     // cars->dump_objects();
-    cars->create_object_with_primary_key("Tesla 10").set(col_price, Decimal128("499999.5"));
+    auto tesla = cars->create_object_with_primary_key("Tesla 10").set(col_price, Decimal128("499999.5"));
     CHECK_EQUAL(stock.size(), 2);
     CHECK_EQUAL(cars->size(), 2);
     CHECK(finn.get<ObjKey>(col_owns));
 
     // cars->dump_objects();
+    tesla.invalidate();
+    CHECK_EQUAL(stock.size(), 1);
+    CHECK_EQUAL(stock.get(0), skoda.get_key());
+    CHECK_EQUAL(cars->size(), 1);
+
+    cars->create_object_with_primary_key("Tesla 10").set(col_price, Decimal128("499999.5"));
+    CHECK_EQUAL(stock.size(), 2);
+    CHECK_EQUAL(cars->size(), 2);
+    CHECK(finn.get<ObjKey>(col_owns));
+}
+
+
+TEST(Links_LinkRedirection)
```

Comment:
I am a bit in doubt what this test is testing that is not tested in the previous test.

Reply:
Yeah, it is redundant